images/macos/tests/System.Tests.ps1

```diff
@@ -39,13 +39,13 @@ Describe "Audio device" -Skip:($os.IsVentura) {
     }
 }
 
-Describe "Screen Resolution" {
+Describe "Screen Resolution" -Skip:($os.IsVenturaArm64) {
     It "Screen Resolution" {
         system_profiler SPDisplaysDataType | Select-String "Resolution" | Should -Match "1176 x 885|1920 x 1080"
     }
 }
 
-Describe "Open windows" {
+Describe "Open windows" -Skip:($os.IsVenturaArm64) {
```

Comment:
skip this test for intel too?

Reply:
@Alexey-Ayupov lets introduce `IsVeertu` check to skip Anka-specific conditions

Reply:
Done